cmd/execution/main.go

```diff
@@ -199,7 +202,7 @@ func main() {
 					collector,
 				)
 
-				blockDataUploader = asyncUploader
+				blockDataUploaders = append(blockDataUploaders, asyncUploader)
```

Comment:
I wonder if maybe instead of slice of uploaders we should create a new uploader which calls multiple ones - `MultiUploader` of sort.